DigitalLearningSolutions.Data.Tests/Services/DiagnosticAssessmentDataServiceTests/DiagnosticAssessmentDataServiceNextTests.cs

```diff
@@ -1,328 +1,328 @@
-        }
-
-        [Test]
-        public void Get_diagnostic_assessment_nextSection_skips_sections_full_of_archived_tutorials()
-        {
-            using (new TransactionScope())
-            {
-                // Given
-                const int candidateId = 210962;
-                const int customisationId = 15062;
-                const int sectionId = 246;
-
-                // The tutorials of what would be the next section, 247;
-                tutorialContentTestHelper.ArchiveTutorial(1136);
-
-                const int expectedNextSectionId = 248;
-
-                // When
-                var result = diagnosticAssessmentDataService.GetDiagnosticAssessment(customisationId, candidateId, sectionId);
-
-                // Then
-                result.Should().NotBeNull();
-                result!.NextSectionId.Should().Be(expectedNextSectionId);
-            }
-        }
-
-        [Test]
-        public void Get_diagnostic_assessment_nextSection_returns_section_with_only_diagnostic_assessment()
-        {
-            // Given
-            const int candidateId = 74411;
-            const int customisationId = 5852;
-            const int sectionId = 150;
-
-            const int expectedNextSectionId = 151; // All tutorials are CustomisationTutorials.Status = 0, though some DiagStatus = 1
-
-            // When
-            var result = diagnosticAssessmentDataService.GetDiagnosticAssessment(customisationId, candidateId, sectionId);
-
-            // Then
-            result.Should().NotBeNull();
-            result!.NextSectionId.Should().Be(expectedNextSectionId);
-        }
-
-        [Test]
-        public void Get_diagnostic_assessment_nextSection_returns_section_with_only_post_learning_assessment()
-        {
-            // Given
-            const int customisationId = 10820;
-            const int candidateId = 1;
-            const int sectionId = 104;
-            const int expectedNextSectionId = 105; // All tutorials are CustomisationTutorials.Status and DiagStatus = 0
-                                                   // Customisations.IsAssessed = 1 and Sections.PLAssessPath is not null
-                                                   // When
-            var result = diagnosticAssessmentDataService.GetDiagnosticAssessment(customisationId, candidateId, sectionId);
-
-            //Then
-            result.Should().NotBeNull();
-            result!.NextSectionId.Should().Be(expectedNextSectionId);
-        }
-
-        [Test]
-        public void Get_diagnostic_assessment_nextSection_skips_assessed_section_with_no_assessment_path()
-        {
-            using (new TransactionScope())
-            {
-                // Given
-                const int customisationId = 10820;
-                const int candidateId = 1;
-                const int sectionId = 104;
-
+﻿namespace DigitalLearningSolutions.Data.Tests.Services.DiagnosticAssessmentDataServiceTests

```

Comment:
What's going on with the large number of supposed changes in this file?

Reply:
Line endings, I did a replace all for //Given, //When and //Then. This file also had mixed line endings so I fixed that.